app/src/main/java/com/release/keyneez/util/extension/ContextExt.kt

```diff
@@ -0,0 +1,24 @@
+package com.release.keyneez.util.extension
+
+import android.app.Activity
+import android.content.Context
+import android.view.View
+import android.view.inputmethod.InputMethodManager
+import android.widget.Toast
+import com.google.android.material.snackbar.Snackbar
+
+/** hide keyboard from window */
+fun Context.hideKeyboard(view: View) {
+    val inputMethodManager = getSystemService(Activity.INPUT_METHOD_SERVICE) as InputMethodManager
+    inputMethodManager.hideSoftInputFromWindow(view.windowToken, 0)
+}
+
+/** Make a Snackbar to display a message for 2 seconds */
+fun Context.showSnackbar(view: View, msg: String) {
+    Snackbar.make(view, msg, Toast.LENGTH_SHORT).show()
+}
+
+/** Make a Toast to display a message for 2 seconds */
+fun Context.showToast(msg: String) {
+    Toast.makeText(this, msg, Toast.LENGTH_SHORT).show()
+}
```

Comment:
```suggestion
/** Make a Snackbar to display a message for 1.5 seconds */
fun Context.showSnackbar(view: View, msg: String) {
    Snackbar.make(view, msg, Toast.LENGTH_SHORT).show()
}

/** Make a Toast to display a message for 1.5 seconds */
fun Context.showToast(msg: String) {
    Toast.makeText(this, msg, Toast.LENGTH_SHORT).show()
}
```
저번 세미나에서 들었는데 1.5초래요 ^_ㅠ